scripts/pi-hole/js/index.js

```diff
@@ -655,10 +655,14 @@ function updateSummaryData(runOnce) {
         }
 
         ["ads_blocked_today", "dns_queries_today", "ads_percentage_today", "unique_clients"].forEach(function(today) {
-            var todayElement = $("span#" + today);
-            todayElement.text() !== data[today] &&
-            todayElement.text() !== data[today] + "%" &&
-            $("span#" + today).addClass("glow");
+            var $todayElement = $("span#" + today);
```

Comment:
@DL6ER is this a single element? If so we could just switch later to `document.querySelector`, `textContent` and `classList.add`

Reply:
Yes. Looking at this code, I seem to recall this coming from the first months after I have first seen JavaScript code in my life. There is likely a lot of room for improvements.

Reply:
Cool, I'll make the changes later in separate patches.